projects/igniteui-angular/src/lib/grids/summaries/grid-summary.service.ts

```diff
@@ -37,8 +38,8 @@ export class IgxGridSummaryService {
             let columnName = args.cellID ? this.grid.columnList.find(col => col.index === args.cellID.columnID).field : undefined;
             if (columnName && this.grid.rowEditable) { return; }
 
-            const isGroupedColumn = this.grid.groupingExpressions &&
-                    this.grid.groupingExpressions.map(expr => expr.fieldName).indexOf(columnName) !== -1;
+            const isGroupedColumn = (this.grid as any).groupingExpressions &&
```

Comment:
Please, don't cast as `any`. You can use the `GridType` interface from `lib/grids/common/grid.interface.ts` as a type.